exercise/part2-school-explorer/index.js

```diff
@@ -131,17 +116,164 @@ be used.
 At this point, if you filter the schools by grade or zip code, you should see
 new items get added to the map and the list. However, the old items are still
 there as well. Update the `updateSchoolMarkers` and `updateSchoolList` functions
-clear the map and the list element before adding new items.
+// clear the map and the list element before adding new items.
+
+LOOP array for update school list?
+filtered schools = (do we define a new array?) => {return looped array}
+update school markers= event handler => {return new markers (def by variable name?)}
+const newSchool = (newSchoolList) => {console.log(newSchoolList)}
 
 ==================== */
 
-let updateSchoolMarkers = (schoolsToShow) => {};
+/* global schools */
+const schoolMap = L.map('school-map').setView([39.95303901388685, -75.16341794003617], 13);
+const schoolLayer = L.layerGroup().addTo(schoolMap);
+L.tileLayer('https://stamen-tiles-{s}.a.ssl.fastly.net/toner/{z}/{x}/{y}{r}.{ext}', {
+  attribution: 'Map tiles by <a href="http://stamen.com">Stamen Design</a>, <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a> &mdash; Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+  subdomains: 'abcd',
+  minZoom: 0,
+  maxZoom: 18,
+  ext: 'png',
+}).addTo(schoolMap);
+const schoolList = document.querySelector('#school-list');
+const gradeLevelSelect = document.querySelector('#grade-level-select');
+const zipCodeSelect = document.querySelector('#zip-code-select');
+/* ====================
+# Exercise: School Explorer
+Saiya notes: above is the way we set the map tile and zoom, same for all hw's can change map tile if i want
```

Comment:
@sighuh A single linter error and it's complaining that this comment line is too long! I'mma let it slide, but totally coulda been fixed.